benchmarks/bulk-insert-and-query.cc

```diff
@@ -731,12 +737,60 @@ int main(int argc, char * argv[]) {
       cout << setw(NAME_WIDTH) << names[a] << cf << endl;
   }
   a = 31;
+//  if (algorithmId == a || algorithmId < 0 || (algos.find(a) != algos.end())) {
+//     auto cf = FilterBenchmark<
+//          VQFilter<uint64_t, SimpleMixSplit>>(
```

Comment:
Hm, it looks like you have commented out the VQFilter... is there a problem with it?

Reply:
Sorry about that. Another workaround fix to try make my cmake work.